modules/platforms/cpp/ignite/client/compute/broadcast_execution.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "ignite/client/compute/job_execution.h"
+#include "ignite/common/ignite_result.h"
+
+#include <vector>
+
+namespace ignite {
+
+/**
+ * Broadcast execution control object, provides information about the broadcast execution process and result.
+ */
+class broadcast_execution {
+public:
+    // Default
+    broadcast_execution() = default;
```

Comment:
Do we need an empty default ctor?

Reply:
I prefer to have them for all non-abstract classes, as without them you can not, for example, store the instances of the class in containers.